queries.yaml

```diff
@@ -273,3 +273,28 @@ post_process:
     params:
       source_layer: roads
       start_zoom: 9
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 14
+      properties: [is_bridge, is_tunnel, oneway]
```

Comment:
These properties are used to calculate the sort key z-order of road features with respect to one another. It's okay to drop them early, as long as the sort key is calculated **before** they are dropped?